index.js

```diff
@@ -681,3 +683,42 @@ function registerShortcuts() {
 function needsCrashReporter(argo) {
     return !!(argo['diagnostics'] || argo['enable-crash-reporting']);
 }
+
+function validatePreloadScripts(options) {
```

Comment:
Any reason this is not in preload_scripts.ts and imported into main?

Reply:
@rdepena I initially put it in `preload_scripts.ts` (as well as the normalization function that is located in `convert_options` now), but importing this/these creates circular dependencies, so had to move it out of `preload_scripts.ts`

Reply:
gotcha